hardware/onewire/onewire.h

```diff
@@ -69,6 +69,12 @@
 #define OW_DEBUG_POLL(...)    ((void) 0)
 #endif
 
+#ifdef DEBUG_ECMD_OW_GET
+#include "core/debug.h"
+#define OW_DEBUG_GET(str...) debug_printf ("OW-GET: " str)
+#else
+#define OW_DEBUG_GET(...)    ((void) 0)
```

Comment:
Oh there were some debug_printf´s in the ecmd code which spamed my syslog should i create a seperate PR for this?